doc/lua_api.md

```diff
@@ -9095,12 +9095,13 @@ Player properties need to be saved manually.
 ```lua
 {
     hp_max = 10,
-    -- Defines the maximum and default HP of the entity
+    -- Defines the maximum and default HP of the object
     -- For Lua entities the maximum is not enforced.
-    -- For players this defaults to `minetest.PLAYER_MAX_HP_DEFAULT`.
+    -- For players this defaults to `core.PLAYER_MAX_HP_DEFAULT` (20).
```

Comment:
I just found this:

https://github.com/minetest/minetest/blob/d4daa9fd40f7a91ad6536ebef10553763cefac13/src/script/cpp_api/s_entity.cpp#L212-L213

I guess the different default value for entities should also be documented here.

Reply:
I'll push a commit to mention the two possible values.
I'm assuming the difference between an Entity and an Object are documented elsewhere? 
Just wondering if the applicable value will be apparent to the person reading this particular documentation.